default.json

```
@@ -116,6 +116,10 @@
       "matchUpdateTypes": ["minor", "patch"],
       "matchCurrentVersion": "!/^v?0/",
       "automerge": true
+    },
+    {
+      "matchRepositories": ["balena-io/balena-ui"],
+      "commitBody": "Update {{depName}}\n\nChangelog-entry: Update {{depName}}\nChange-type: patch"
```

I guess it could even be v. I mean there is no need to use the Changelog-entry at all if we do not mention the `from to` versions at the first place.
```suggestion
      "commitBody": "Update {{depName}}\n\nChange-type: patch"
```

---

The CHANGELOG will use the commit title if we don't include a Changelog-entry, and the commit title has the version. See [this](https://balena.zulipchat.com/#narrow/stream/345752-balena-io.2Finternal-tooling/topic/Renovate.20major.20change-types) thread.

---

Oh you are right, I was confused. In that case I guess we don't need the `Update {{depName}}` part, right?
```suggestion
      "commitBody": "Changelog-entry: Update {{depName}}\nChange-type: patch"
```

---

Actually I just realized we need the version in the [body](https://docs.renovatebot.com/configuration-options/#commitbody) in order for embedded changelogs to work. Let me push a change.

---

@klutchell I think we shouldn't do that for the UI, since nested changelogs would expose version numbers which we want to avoid

---

commit and message have been updated

---

Yes, we want nested changelogs disabled in the UI, since we don't want the versions to show up to users.

---

I confirm @thgreasi's above message